packages/jest-scheduler/src/__tests__/JestScheduler-test.js

```diff
@@ -0,0 +1,164 @@
+import {
+  unstable_getCurrent as getCurrent,
+  unstable_subscribe as subscribe,
+  unstable_trace as trace,
+  unstable_wrap as wrap,
+} from 'scheduler/tracing';
+import {createMockSubscriber} from 'jest-scheduler';
+
+describe('JestScheduler', () => {
+  if (__DEV__) {
+    it('should work with an empty set of interactions', () => {
+      // Sanity test.
+      expect(getCurrent()).toHaveBeenTracedWith([]);
+    });
+
+    it('should trace interactions through callbacks', done => {
+      function onFinished(value) {
+        expect(value).toBe('something');
+
+        // Verify that the callback has been called with the expected interactions.
```

Comment:
I added inline comments initially, as I wrote these tests, thinking I might point people to the test as a reference in lieu of docs– but maybe I should remove them?